allennlp/training/metrics/auc.py

```diff
@@ -52,7 +52,7 @@ def __call__(self,
             raise ConfigurationError("AUC can be used for binary tasks only. gold_labels has {} unique labels, "
                                      "expected at maximum 2.".format(unique_gold_labels.numel()))
 
-        gold_labels_is_binary = list(torch.sort(unique_gold_labels)[0].numpy()) == [0, 1]
```

Comment:
I think this would be cleaner as

```
set(unique_gold_labels.tolist()) <= {0, 1}
```

Reply:
sorry, for the delay, this slipped through the cracks

Reply:
I thought I forgot to push updated codes.
Sorry, why there is index [0] in the original code?

Reply:
`torch.sort` returns a tuple (sorted_values, sorted_indices), but the original code only needed the value.

if we're using `set` we don't need to sort at all

Reply:
Sorry, I forget the `sort`. Was thinking about `torch.unique`. I updated with `set` and leave `torch.unique` there since reducing with CUDA and interfacing with only a small tuple seems more efficient.